elasticdl/pkg/ps/server.go

```diff
@@ -63,23 +71,53 @@ func createMasterClient(masterAddr string) *MasterClient {
 }
 
 // NewServer creates a Server instance
-func NewServer(ID int, optType string, optArgs string, masterAddr string, evaluationSteps int) *Server {
+func NewServer(ID int, optType string, optArgs string, masterAddr string,
```

Comment:
Rename `NewServer`, maybe `ServerFactory`?

Reply:
It's the naming convention of Go, we usually use `NewStruct` to create a Struct instance.